### PR TITLE
feature(SctRunner): add kubectl bin

### PR DESCRIPTION
**Pull request by KnifeyMoloko**
Adding the `kubectl` binary to our base SctRunner classes `prereqs_script` so that we have an additional tool for kubernetes deployments debugging.

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- ~~[ ] I added the relevant `backport` labels~~
- ~~[ ] New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [x] All new and existing unit tests passed (CI)
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

File: sdcm/sct_runner.py [hunk 1]
```diff
@@ -104,10 +107,10 @@ def install_prereqs(self, public_ip):
             # Jenkins pipelines run /bin/sh for some reason
             unlink /bin/sh
             ln -s /bin/bash /bin/sh
-
+            echo "cd ~/home/ubuntu" >> /home/ubuntu/.bashrc
```
Comment (vponomaryov) [inline sdcm/sct_runner.py:110]: Why this is needed? And how is it related to the addition of "kubectl"?

File: sdcm/sct_runner.py [hunk 2]
```diff
@@ -104,10 +107,10 @@ def install_prereqs(self, public_ip):
...
         """)
         remoter = self.get_remoter(host=public_ip)
-        result = remoter.run(f"sudo bash -cxe '{prereqs_script}'", ignore_status=True)
+        result = remoter.run(f"sudo bash -cxe '{prereqs_script}'", ignore_status=False)
```
Comment (vponomaryov) [inline sdcm/sct_runner.py:113]: This one looks unrelated to addition of "kubectl".
If we need it, then better to make it be separate commit in the same PR with description why it is done.

**Review by vponomaryov — CHANGES_REQUESTED** [review]

File: sdcm/sct_runner.py (hunk 2 above)
Comment (KnifeyMoloko) [reply sdcm/sct_runner.py]: Not needed and now reverted to `True`

File: sdcm/sct_runner.py (hunk 1 above)
Comment (KnifeyMoloko) [reply sdcm/sct_runner.py]: Not needed - removed now.

File: sdcm/sct_runner.py [hunk 3]
```diff
@@ -27,7 +27,7 @@ class ImageType(Enum):
 
 class SctRunner:
     """Provisions and configures the SCT runner"""
-    VERSION = 1.2  # Version of the Image
+    VERSION = 1.4  # Version of the Image
```
Comment (vponomaryov) [inline sdcm/sct_runner.py:30]: Curious why it is not 1.3
But not a blocker for me.

**Review by vponomaryov — DISMISSED** [review]
LGTM

File: sdcm/sct_runner.py (hunk 3 above)
Comment (KnifeyMoloko) [reply sdcm/sct_runner.py:30]: We have image with 1.3 already created in eu-west-2 region (not sure why), so I wanted to avoid overwriting that :)

Comment (vponomaryov) [reply sdcm/sct_runner.py:30]: @bentsi , is it ok? ^

**Review by vponomaryov — APPROVED** [review]

File: sdcm/sct_runner.py (hunk 3 above)
Comment (bentsi) [reply sdcm/sct_runner.py:30]: yes

**Comment by bentsi** [comment]
please create the images in all regions that we support:

- eu-west-1
- eu-west-2
- eu-north-1
- eu-central-1
- us-east-1

**Comment by KnifeyMoloko** [comment]
> please create the images in all regions that we support:
> 
> * eu-west-1
> * eu-west-2
> * eu-north-1
> * eu-central-1
> * us-east-1

Done.

**Review by bentsi — APPROVED** [review]
